x/accounts/defaults/base/go.mod

```diff
@@ -1,6 +1,6 @@
 module cosmossdk.io/x/accounts/defaults/base
```

Comment:
do accounts need to be modified with the new collections changes?

Reply:
no, this was go-mod-tidy-all :/